packages/modules/src/modules/banners/choiceCardsButtonsBanner/components/ChoiceCardAmountButtons.tsx

```diff
@@ -43,15 +45,13 @@ const choiceCardsContainer = css`
     }
 `;
 
-const choiceCardOrOtherAmountContainer = css`
+const choiceCardOrOtherAmount = (isAmountMissing: boolean, backgroundColour?: string) => css`
     margin-bottom: ${space[1]}px;
-    ${from.mobileLandscape} {
-        margin-bottom: ${space[3]}px;
-    }
-`;
+    ${isAmountMissing ? `` : `${from.mobileLandscape} { margin-bottom: ${space[3]}px; }`}
 
-const choiceCardOrOtherAmountMissing = css`
-    margin-bottom: ${space[1]}px;
+    > label {
+        background-color: ${backgroundColour ?? ''};
```

Comment:
nit-picky, but the initial value of background-color is transparent. Would it be better to use that as the right hand side of the ternary instead of a blank string:

eg

`background-color: ${backgroundColour ?? 'transparent'};`

Reply:
Yep good idea to make explicit, thanks, changed